tests/foreman/cli/test_host.py

```diff
@@ -1014,12 +1015,20 @@ def test_positive_create_using_libvirt_without_mac(self):
 
         :CaseImportance: Critical
         """
+        new_loc = make_location()
```

Comment:
I would move that to setup also in a same way we have `new_org`

Reply:
Yep, was thinking about it but was afraid :) Moved it there and it still works.

Reply:
Thank you!